integration-tests/update-local-embrace.sh

```diff
@@ -40,14 +58,17 @@ if [ "$test_harness_only" = false ]; then
   ./pack.sh ../packages/react-native-tracer-provider/ artifacts/embrace-io-react-native-tracer-provider-local.tgz
 fi
 
-# build and pack the test harness
-npm --prefix test-harness run build
-./pack.sh  test-harness/ artifacts/embrace-io-react-native-test-harness-local.tgz
-
-# update all test apps with the sdk packages and the test harness
-# TODO EMBR-4923 put test apps within a common folder and use lerna to run this command across all of them
-npm --prefix basic-test-app add \
-  ./artifacts/embrace-io-react-native-local.tgz \
-  ./artifacts/embrace-io-react-native-tracer-provider-local.tgz \
-  ./artifacts/opentelemetry-instrumentation-react-native-navigation-0.1.0.tgz \
-  ./artifacts/embrace-io-react-native-test-harness-local.tgz
\ No newline at end of file
+if [ "$skip_test_harness" = false ]; then
+  # build and pack the test harness
+  npm --prefix test-harness run build
+  ./pack.sh  test-harness/ artifacts/embrace-io-react-native-test-harness-local.tgz
+fi
+
+if [ "$skip_sdk_packages" = false ] || [  "$skip_test_harness" = false ]; then
```

Comment:
let me know if I'm wrong but if I pass --skip-sdk-packages (so `skip_sdk_packages` is true) and do not pass --skip-test-harness (`skip_test-harness` is false) then the script will still try to install everything, right?

but since `skip_sdk_packages` is `true` the sdk packages won't be packed, so there will not be there. will this throw an error? (same thing would apply if I pass --skip-test-harness and don't pass --skip-sdk-packages)

Reply:
updated